language/move-lang/tests/move_check/expansion/public_script_main.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/expansion/public_script_main.move:2:1 ───
+   │
+ 2 │ public(script) fun main() {
+   │ ^^^^^^^^^^^^^^ Extraneous 'public(script)' modifier. Script functions are always public(script)
```

Comment:
Did you forget to update baseline? In the fmt string I see 'public(script)' but don't see it here

Reply:
Yes, I forgot this case.. And this is also the reason why the unit test in CI is not passing :( Will be updated.